auth/app/src/main/java/com/google/firebase/quickstart/auth/PhoneAuthActivity.java

```diff
@@ -67,6 +68,8 @@ public void onVerificationFailed(@NonNull FirebaseException e) {
                     // Invalid request
                 } else if (e instanceof FirebaseTooManyRequestsException) {
                     // The SMS quota for the project has been exceeded
+                } else if (e instanceof FirebaseAuthMissingActivityRecaptchaException) {
+                    // Recaptcha verification attempted with invalid acitvity
```

Comment:
Maybe say "with null activity"?
```suggestion
                    // reCAPTCHA verification attempted with null Activity
```